src/transformers/generation_utils.py

```diff
@@ -661,6 +677,8 @@ def generate(
                 sequences.
             no_repeat_ngram_size (:obj:`int`, `optional`, defaults to 0):
                 If set to int > 0, all ngrams of that size can only occur once.
+            encoder_no_repeat_ngram_size (:obj:`int`, `optional`, defaults to 0):
+                If set to int > 0, all ngrams of that size can only occur within the encoder_input_ids they are discarded from decoder.
```

Comment:
```suggestion
                If set to int > 0, all ngrams of that size that occur in the `encoder_input_ids` cannot occur in the `decoder_input_ids`
```
if I understood correctly